demo/examples/src/wallet_connect.cc

```diff
@@ -153,6 +163,52 @@ int main(int argc, char *argv[]) {
                       << bytes_to_hex_string(receipt).c_str() << std::endl;
         }
 
+        // send contract transaction
+        if (test_erc20) {
+            WalletConnectTxCommon common;
+            // Verify the contract
+            // Test ERC20 Token: GLD
+            // https://testnet.cronoscan.com/token/0xc213a7b37f4f7ec81f78895e50ea773aa8e78255
+            Erc20 erc20 =
+                new_erc20("0xC213a7B37F4f7eC81f78895E50EA773aA8E78255",
+                          "https://evm-t3.cronos.org", 338);
+            assert(erc20.name() == "Gold");
+            assert(erc20.symbol() == "GLD");
+            assert(erc20.decimals() == 18);
+            rust::String from_address = rust::String(
+                std::string("0x") +
+                address_to_hex_string(result.addresses[0].address).c_str());
+            U256 erc20_balance = erc20.balance_of(from_address);
+            std::cout << "erc20 balance=" << erc20_balance.to_string()
+                      << std::endl;
+
+            // construct tx info
+            rust::String contract_action =
+                R"({
+                      "Erc20Transfer": {
```

Comment:
also if json breaks, spec of contract, make it return detail error

Reply:
Yup, it is already detailed enough, since the contract_action json is decoded by serde crate.